shadow-dom/untriaged/events/events-created-by-users-do-not-stop/test-001.html

```diff
@@ -42,18 +42,17 @@
     s.appendChild(inp1);
 
     s.addEventListener('abort', A_05_04_01_T01.step_func(function(event) {
-    	assert_equals(event.target.getAttribute('id'), 'inp1', 'Inside shadoe tree: Wrong target');
+    	assert_equals(event.target.getAttribute('id'), 'inp1', 'Inside shadow tree: Wrong target');
     }), false);
 
     d.body.addEventListener('abort', A_05_04_01_T01.step_func(function(event) {
-    	assert_true(false, 'Abort event should always be stopped at Shadow boundary');
-    }), false);
+        assert_true(true, 'Abort event should not be stopped at Shadow boundary if created by users');
+        A_05_04_01_T01.done();
+    }), true);
```

Comment:
What was the original intention to change the 3rd parameter for addEventListener() from false to true?

Setting it to true means that, in the capturing phaase the event handler for `d.body` is called first, then
in the bubbling phase the event handler for `s` is called.  Therefore, calling `done()` may prematurely
finish this test without guaranteeing that the event handler for `s` is called.

If the capturing phase in unintentional, how about
1. change `d.body` handler run in bubbling phase (`true` -> `false`)
2. introduce a flag (e.g. `var pass = false;`) on line 43
3. set the flag `true` after line 45, at the end of the handler for `s`
4. check the flag in the handler for `d.body` (line 49) as `assert_true(pass, 'Abort event should ...`
?

(The same comment applies to all remaining tests in this PR)